provider-directory/src/main/java/gov/va/api/health/providerdirectory/service/controller/practitioner/PractitionerController.java

```diff
@@ -94,11 +70,49 @@ public PractitionerController(
             Practitioner.Bundle::new));
   }
 
+  /** If a user were to search by a parameter other then family or given, the call would
+   * be checked and failed earlier at the PPMS call. */
+  private PractitionerWrapper search(MultiValueMap<String, String> parameters) {
+    ProviderResponse providerResponse;
+    if (parameters.containsKey("identifier")) {
+      String identifier = parameters.get("identifier").get(0);
+      providerResponse = client.providersForId(identifier);
+    } else if (parameters.containsKey("name")) {
+      String name = parameters.get("name").get(0);
+      providerResponse = client.providersForName(name);
+    } else {
+      String familyAndGiven =
+              parameters.get("family").get(0)
+                      + ", "
+                      + parameters.get("given").get(0);
+
+      providerResponse = client.providersForName(familyAndGiven);
+
+    }
+    String providerIdentifier = providerResponse.value().get(0).providerIdentifier().toString();
+
+    ProviderContactsResponse providerContactsResponse = providerContact(providerIdentifier);
+
+    PractitionerWrapper root =
+            PractitionerWrapper.builder()
+                    .providerContactsResponse(providerContactsResponse)
+                    .providerResponse(providerResponse)
+                    .build();
+    return root;
+  }
+
   @SneakyThrows
-  private ProviderContacts ppmsProviderContact(String id) {
+  private ProviderContactsResponse providerContact(String id) {
```

Comment:
This one-line method doesn't seem very useful... let's inline it where it was used.